src/MessageUI/MessageUI.cs

```diff
@@ -12,22 +12,19 @@
 
 namespace XamCore.MessageUI {
 
-	// untyped enum -> MFMailComposeViewController.h
```

Comment:
there's no reason to remove the comments, they make it clear they should not be `[Native]`